test/verify/check-packagekit

```diff
@@ -383,32 +554,22 @@ class TestUpdates(NoSubManCase):
         b.wait_visible("#app div.pf-c-progress__bar")
 
         # should have succeeded and show restart
-        b.wait_in_text("#app .pf-c-empty-state h1", "Restart recommended")
-        b.wait_visible("#app .pf-c-empty-state button.pf-m-link:Contains('Ignore')")
-
-        # new versions are now installed
-        m.execute("test -f /stamp-norefs-bin-2-1 && test -f /stamp-norefs-doc-2-1")
+        b.wait_visible(".pf-c-empty-state .pf-c-title:contains('Update was successful')")
+        b.wait_visible("#ignore")
 
         # history on restart page should show the three non-security updates
         b.click(".expander-title button")
-        assertHistory("ul", ["buggy", "norefs-bin", "norefs-doc"])
+        self.assertHistory("ul", ["buggy", "norefs-bin", "norefs-doc"])
 
-        # do the reboot; this will disconnect the web UI
-        b.click("#app .pf-c-empty-state button.pf-m-primary")
-        b.switch_to_top()
-        b.wait_in_text(".curtains-ct h1", "Disconnected")
-
-        # ensure that rebooting actually worked
-        m.wait_reboot()
-        m.start_cockpit()
-        b.reload()
-        b.login_and_go("/updates")
```

Comment:
Please put that back, conditionalized on "OS does not support tracer". It's the only place in our tests that validates the reboot workflow for OSes without tracer.